app.py

```
@@ -1,17 +1,29 @@
+
 app=Flask(__name__)
 
-@app.route("/")
+userValidation=UserValidation()
+userManipulation=UserManipulation()
+
+
+@app.route("/",methods=['POST','GET'])
 def index():
     if request.method=='POST':
         user_data=dict(request.form) if request.form else request.get_json()
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)